_posts/en/newsletters/2021-04-21-newsletter.md

```diff
@@ -0,0 +1,247 @@
+  (90%) Taproot softfork. Starting with the first difficulty
+  adjustment after April 24th, miners will be able to signal readiness
+  for Taproot activation on bit 2. If 1815 (90%) of one difficulty
+  period's 2016 blocks in the signaling window signal readiness, the
+  softfork activation locks in. The signaling window ends with the first
+  difficulty adjustment after August 11th. If locked in, Taproot will be
+  activated at block height 709632 which is expected to be reached
+  around November 12th.
+
+- [Bitcoin Core #21602][] updates the `listbanned` RPC with two
+  additional fields: `ban_duration` and `time_remaining`.
+
+- [C-Lightning #4444][] adds [lnprototest][] (LN Protocol Tests) to the
+  default targets for C-Lightning's Continuous Integration (CI) tests
+  and also makes it easier for developers to run the tests from
+  C-Lightning's usual build system.  The LN protocol tests make it easy
+  for an implementation to test that they're following the [LN protocol
+  specification][].
+
+- [LND #4588][] skips creating a change output in cases where the amount
+  of change is so small that it's worth less than the amount it would
+  cost to spend it.
+
+- [LND #5193][] disables channel validation by default for LND instances
+  using the Neutrino client (which implements the [compact block
+  filters][topic compact block filters] protocol).  This option assumes
+  that channel advertisements received from a peer are correct, saving
+  the client from having to download any old blocks necessary to verify
+  those advertisements.  This has the downside that payment attempts
+  made using falsely advertised channels will fail, wasting time but not
+  causing the loss of money---a reasonable tradeoff for anyone already
+  choosing to use a lightweight client. This new default behavior may be
+  disabled by using the new configuration option
+  `--neutrino.validatechannels=true`.
+
+- [LND #5154][] adds basic support for using LND with a pruned full
+  node, allowing LND to externally request from other Bitcoin nodes a
+  block that has been deleted by the local node.  LND can then extract
+  whatever information it needs from the block without going through the
+  pruned node.  Because the user's own full node previously verified the
+  block, this doesn't change the security model.
+
+- [LND #5187][] adds new `channel-commit-interval` and
+  `channel-commit-batch-size` parameters that can be used to configure
+  how long LND will wait before sending a channel state update and the
+  maximum number of changes it'll send in one update.  The higher these
+  values are, the more efficient a busy LND node will be, although that
+  efficiency comes at the cost of having a slightly higher latency.
+
+- [Rust-Lightning #858][] adds internal support for interoperating with
+  Electrum-style blockchain data sources where transactions are confirmed and
+  unconfirmed for reorgs rather than blocks being connected and disconnected.
```

Comment:
The way this currently reads to me is "(confirmed and unconfirmed) for reorgs". I think what you meant was "confirmed [for connected blocks] and unconfirmed for disconnected blocks (reorgs)".

I think you can just remove this whole part:

```suggestion
  Electrum-style blockchain data sources.
```

Describing how the electrum protocol delivers updates to clients seems too low level.